clear_cache/management/commands/clear_cache.py

```diff
@@ -1,16 +1,40 @@
 from django.conf import settings
-from django.core.cache import cache
+from django.core.cache import cache, caches, InvalidCacheBackendError
 from django.core.management.base import BaseCommand, CommandError
 
 
 class Command(BaseCommand):
     """A simple management command which clears the site-wide cache."""
-    help = 'Fully clear your site-wide cache.'
-
+    help = 'Fully clear your site-wide cache. No arguments will clear the default cache.'
+    
+    def add_arguments(self, parser):
+        # Positional arguments
+        parser.add_argument('cache_names', nargs='*', type=str, help='Name of a specific cache to clear')
+        
+        # Named (optional) arguments
+        parser.add_argument('--all', action='store_true', help='Clear all caches')
+    
     def handle(self, *args, **kwargs):
+        cache_names = kwargs.get('cache_names')
+        delete_all = kwargs.get('all')
+        
+        if not cache_names:
+            cache_names = ['default']
+        
         try:
             assert settings.CACHES
-            cache.clear()
-            self.stdout.write('Your cache has been cleared!\n')
         except AttributeError:
-            raise CommandError('You have no cache configured!\n')
+            raise CommandError(self.style.WARNING('You have no cache configured!\n'))
+        
+        if delete_all:
+            cache_names = []
+            for name in settings.CACHES:
+                cache_names.append(name)
+        
+        for cache_name in cache_names:
+            try:
+                cache = caches[cache_name]
+                cache.clear()
```

Comment:
It is worth pointing out that `cache.clear()` issues a `SCAN 0 MATCH *` against the selected DB, based on the `LOCATION` of the django cache.   This means it will find all keys for cache aliases that share the `LOCATION`.

For example - let's say I have a brand new redis with no data in it, and i have two caches (c1, c2) defined with the same location.

```
>>> caches['c1'].set('foo', 'bar')
True
>>> caches['c2'].set('foo', 'bar')
True
>>> caches['c2'].clear()
2
```

from the redis `MONITOR`:

```
+1620157208.407551 [9 172.20.0.1:62058] "SCAN" "0" "MATCH" "*"
+1620157208.409341 [9 172.20.0.1:62058] "DEL" "c1:None:foo"
+1620157208.411038 [9 172.20.0.1:62058] "DEL" "c1:None:foo"
```


Does this mean this PR is wrong?  no, but `cache[<alias>].clear()` does more than what you'd expect.